xbmc/utils/ContentUtils.cpp

```diff
@@ -44,3 +44,18 @@ const std::string ContentUtils::GetPreferredArtImage(const CFileItem& item)
   }
   return item.GetArt("thumb");
 }
+
+CFileItem* ContentUtils::GeneratePlayableTrailerItem(const CFileItemPtr& item,
+                                                     const std::string& label)
+{
+  CFileItem* trailerItem = new CFileItem();
```

Comment:
you should mention in the header doxy that the caller is responsible for deleting this item.